AdobeStockAsset/Model/AssetRepository.php

```diff
@@ -132,8 +132,6 @@ public function getList(SearchCriteriaInterface $searchCriteria): AssetSearchRes
             $searchResults->setTotalCount($collection->getSize());
 
             return $searchResults;
-        } catch (LocalizedException $exception) {
-            throw $exception;
         } catch (\Exception $exception) {
             $this->logger->critical($exception);
             $message = __('An error occurred during get asset list: %error', ['error' => $exception->getMessage()]);
```

Comment:
Let's exclude the system message from displaying in the admin panel
```suggestion
            $message = __('Could not retrieve assets.');
```

Also, I think we should use LocalizedException instead of IntegrationException (as this functionality is not integration)